### PR TITLE
Component for Pagination Bullets

### PR DESCRIPTION
**Pull request by nickeyvee**
A response to #848 

Component can be viewed within storybook.

screenshot 140

### CODE REVIEW

File: client/app/bundles/shared/components/Pagination/Pagination.jsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import css from './Pagination.scss';
+
+export default class Pagination extends React.Component {
+
+   constructor(props) {
```
Comment (andy5995) [inline client/app/bundles/shared/components/Pagination/Pagination.jsx]: Hi @nickeyvee . Indentation needs to be 2 spaces here, not 3. That's one of the reasons the Circle code integration is failing. Other errors can be seen here: https://circleci.com/gh/ifmeorg/ifme/2393?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

**Comment by andy5995** [comment]
Thank you for working on this, @nickeyvee :) I left a comment. The details of the code integration test I mentioned can be viewed on the main page of your pull pull request. People don't get notified automatically when the CI test fails, so just wanted to give you a heads-up. Cheers! :)

**Comment by nickeyvee** [comment]
@andy5995 Thanks for letting me know, I'll go ahead and see if I can address these.

**Comment by prateksha** [comment]
@nickeyvee Please fix the circleci tests and update your PR. If you are busy, can I work on it?

**Comment by nickeyvee** [comment]
@prateksha I haven't been able to get the test suite to run locally on my machine so I can fix my errors. If you want you can take over this issue.

**Pull request closed without merging**